cmd/awscollector/Dockerfile

```diff
@@ -16,20 +16,9 @@ ENV PATH="/usr/local/go/bin:${PATH}"
 
 # download go modules ahead to speed up the building
 WORKDIR /workspace
-COPY go.mod /workspace/go.mod
-COPY go.sum /workspace/go.sum
+COPY . /workspace
 RUN go mod download
 
```

Comment:
can we keep cherry-picking way of copying and move `go mod download` to the end of `cp`?